skaffold.yaml

```diff
@@ -84,5 +84,12 @@ deploy:
         wait: true
         chartPath: charts/admission
         setValueTemplates:
-          runtime.image.repository: '{{.IMAGE_REPO_europe_docker_pkg_dev_gardener_project_public_gardener_extensions_registry_cache_admission}}'
-          runtime.image.tag: '{{.IMAGE_TAG_europe_docker_pkg_dev_gardener_project_public_gardener_extensions_registry_cache_admission}}@{{.IMAGE_DIGEST_europe_docker_pkg_dev_gardener_project_public_gardener_extensions_registry_cache_admission}}'
+          global.image.repository: '{{.IMAGE_REPO_europe_docker_pkg_dev_gardener_project_public_gardener_extensions_registry_cache_admission}}'
+          global.image.tag: '{{.IMAGE_TAG_europe_docker_pkg_dev_gardener_project_public_gardener_extensions_registry_cache_admission}}@{{.IMAGE_DIGEST_europe_docker_pkg_dev_gardener_project_public_gardener_extensions_registry_cache_admission}}'
```

Comment:
You probably want to adapt the image names to start with `local-skaffold/`.
https://github.com/gardener/gardener/pull/9782#discussion_r1604753127